src/ogd/games/THERMOVR/features/PhasesReached.py

```diff
@@ -0,0 +1,33 @@
+from typing import Any, List
+from ogd.core.extractors.Extractor import ExtractorParameters
+from ogd.core.extractors.features.SessionFeature import SessionFeature
+from ogd.core.schemas.Event import Event
+from ogd.core.schemas.ExtractionMode import ExtractionMode
+from ogd.core.schemas.FeatureData import FeatureData
+
+class PhasesReached(SessionFeature):
+
+    def __init__(self, params: ExtractorParameters):
+        self.phases_reached = set()
+        super().__init__(params=params)
+
+    @classmethod
+    def _getEventDependencies(cls, mode: ExtractionMode) -> List[str]:
+        return ["all_events"]
+
+    @classmethod
+    def _getFeatureDependencies(cls, mode: ExtractionMode) -> List[str]:
+        return []
+
+    def _extractFromEvent(self, event: Event) -> None:
+        game_state = event.EventData.get("all_events")
```

Comment:
There's nothing inside the EventData column called `"all_events"`, so this will always be `None`. The `"all_events"` is just a hardcoded thing we use in `_getEventDependencies` to say we want every event to show up in `_extractFromEvent`.

We can remove this line and the next, and just directly use `event.GameState.get("region")`. There's no other check to make.